wsbroadcastserver/utils.go

```diff
@@ -71,7 +77,9 @@ func ReadData(ctx context.Context, conn net.Conn, earlyFrameData io.Reader, idle
 	// Remove timeout when leaving this function
 	defer func(conn net.Conn) {
 		err := conn.SetReadDeadline(time.Time{})
-		logError(err, "error removing read deadline")
+		if err != nil && !strings.Contains(err.Error(), "use of closed network connection") {
+			logError(err, "error removing read deadline")
```

Comment:
Doesn't logError already have this if statement?

Reply:
:+1: